roles/idrac_user/vars/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+# vars file for idrac_user
+idrac_user_delegate:
+  "{{ lookup('ansible.builtin.env', 'RUNON', default='localhost') }}"
+idrac_user_uri_headers:
```

Comment:
If we have used uri_module in only one place through out the role, I feel we should not populate vars/main.yml with these values, we can simply pass these value directly there.

Reply:
Lets use vars/main.yml itself as its specifically for vars. Also we might enhance few things in module and role later.